packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -80,14 +119,54 @@ describe('convert-runtime-to-plugin', () => {
       },
     });
 
+    const routesManifest = JSON.parse(output['routes-manifest.json']);
+    expect(routesManifest).toMatchObject({
+      version: 3,
+      pages404: true,
+      redirects: [],
+      rewrites: [
+        /* TODO: `handle: miss`
```

Comment:
Currently, zero config routes have a `handle: miss` phase thats used to add a write so that both the extension and the extensionless api file work.

However, the current `routes-manifest.json` doesn't support this so we'll need to provide a new prop in the manifest if we want to keep the same behavior.